diff --git a/node_modules/expo/node_modules/@expo/cli/build/src/start/server/middleware/ExpoGoManifestHandlerMiddleware.js b/node_modules/expo/node_modules/@expo/cli/build/src/start/server/middleware/ExpoGoManifestHandlerMiddleware.js
index 7d211cb..da94de1 100644
--- a/node_modules/expo/node_modules/@expo/cli/build/src/start/server/middleware/ExpoGoManifestHandlerMiddleware.js
+++ b/node_modules/expo/node_modules/@expo/cli/build/src/start/server/middleware/ExpoGoManifestHandlerMiddleware.js
@@ -156,7 +156,9 @@ class ExpoGoManifestHandlerMiddleware extends _ManifestMiddleware.ManifestMiddle
             codeSigningInfo
         });
         const expoUpdatesManifest = {
-            id: _crypto().default.randomUUID(),
+            id: _crypto().default.randomUUID({
+                disableEntropyCache: true
+            }),
             createdAt: new Date().toISOString(),
             runtimeVersion,
             launchAsset: {
